Sources/Build/ManifestBuilder.swift

```diff
@@ -268,12 +285,147 @@ extension LLBuildManifestBuilder {
                         args: arguments
                     )
                 }
-             }
+            }
+        } catch {
+            fatalError("\(error)")
```

Comment:
Same question: seems like we should be able to `throw` out of here? Fatal errors should be due to logic errors in the code, only.